components/brave_wallet_ui/components/shared/create-site-origin/index.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react'
+import { OriginInfo } from '../../../constants/types'
+
+export interface Props {
+  originInfo: OriginInfo
+}
+
+const CreateSiteOrigin = (props: Props) => {
+  const { originInfo } = props
```

Comment:
destructure `eTldPlusOne` & `origin` from `originInfo` for better memo perf

Reply:
Done!